### DISCUSSION TITLE
WoW won't start after installing

### DISCUSSION THREAD

**Discussion by MacGuini** [ogri-la/wowman#412] [Q&A]
I successfully installed Stongbox. Had to install FUSE to get it to work. But since I've done that, I can't launch WoW anymore using Lutris. Anyone else have this issue?

**Comment by torkus** [comment]
hi @MacGuini , I can help with Strongbox problems but not much with Lutris problems. 

Can you tell me more details about your setup (linux distribution, type of filesystem running strongbox and/or wow, etc) as well as what you did to install Strongbox and how you typically launch WoW (packages installed, guides followed, etc)? The more I know the better I can help figure out the state your machine is in.

**Comment by MacGuini** [comment]
I actually just got it working. Checked the logs and a few files were missing. Not sure if it's FUSE related or not. But I was able to get the missing files in order. Maybe a weird coincidence.
